netbox/dcim/filtersets.py

```diff
@@ -1795,6 +1795,10 @@ class InventoryItemFilterSet(DeviceComponentFilterSet, NetBoxModelFilterSet):
         to_field_name='slug',
         label=_('Role (slug)'),
     )
+    device_status = django_filters.MultipleChoiceFilter(
```

Comment:
We can move this to `DeviceComponentFilterSet`, to ensure it's supported for _all_ device components (console ports, interfaces, etc.).

Reply:
Ah, good call, I was going to ask if there was a central place where this stuff can live so we aren't redundantly implementing it on every similar object type.